recipes-core/images/mistysom-image.bbappend

```diff
@@ -49,7 +48,7 @@ IMAGE_INSTALL_append = " wpa-supplicant"
 
 # WiFi Module Laird Sterling-LWB5+ (Disable other WiFi modules because this module compiles a different cfg80211.ko)
 IMAGE_INSTALL_append = " lwb5plus-sdio-div-firmware"
-IMAGE_INSTALL_append = " kernel-module-lwb5p-backports-laird"
+IMAGE_INSTALL_append = " kernel-module-lwb5p-backports-summit"
```

Comment:
@matinlotfali were you able to get the Laird module working, eventually? What's the story here?

Reply:
Just fixed it!

Reply:
Yes I did. I will write details.